v2/cmd/samples/http/sender/main.go

```diff
@@ -30,6 +30,7 @@ func main() {
 			"message": "Hello, World!",
 		})
 
+		ctx = cloudevents.WithEncodingStructured(ctx)
```

Comment:
Is it a leftover from a debugging session?

Reply:
It kinda is, I should switch it to a flag. I used this to send events to azure event grid but it needed more...

Reply:
sure, and please update the documentation of the example here: https://github.com/cloudevents/sdk-go/blob/b3a2bcf1ef60d18bfb312acd5802bffdf46a82ad/v2/cmd/samples/README.md

Reply:
I will revert for now and add later.